### ISSUE TITLE
Use the postcss-selector-matches

### ISSUE THREAD

**Issue by yisibl** [postcss/postcss-custom-selectors#16]
In order to support more complex cases, We can use  the postcss-selector-matches to transform.

@MoOx Do you feel that you are directly dependent on another plugin?

```css
/* input: */
@custom-selector :--any-link :link, :visited;
@custom-selector :--heading h1, h2, h3;

:--heading > a:--any-link {
  color: red;
}

/* Then use the postcss-selector-matches to transform: */
:matches(h1, h2, h3) > a:matches(:link, :visitedk) {
  color: red;
}

/* output: */
h1 > a:link, h2 > a:link, h3 > a:link, h1 > a:visited, h2 > a:visited, h3 > a:visited {
  color: red;
}
```

**Comment by MoOx** [comment]
What is the purpose of this issue ?

The only real issue I can see right is that postcss-custom-selectors is not compatible with postcss-selector-matches and I already opened this issue #15

**Comment by MoOx** [comment]
I think we should close this issue. No ?

**Comment by MoOx** [comment]
Why do you want to add this plugin here ?
You just need to pipe those 2 plugins.
You can see it's already working if you do that on http://cssnext.io/playground/
You should definitely close this issue

**Comment by MoOx** [comment]
Each plugin have their own responsibility.

**Comment by MoOx** [comment]
Closed by #27
